src/tools/compiletest/src/main.rs

```diff
@@ -18,7 +18,7 @@ fn main() {
 
     let config = Arc::new(parse_config(env::args().collect()));
 
-    if !config.has_tidy && config.mode == Mode::Rustdoc {
+    if !config.has_html_tidy && config.mode == Mode::Rustdoc {
         eprintln!("warning: `tidy` is not installed; diffs will not be generated");
```

Comment:
Nit: Maybe disambiguate this warning as well?

Reply:
This is why this PR is a follow-up heh, because https://github.com/rust-lang/rust/pull/131941 is fixing the warning, which is why I noticed and am sending this follow-up.

Reply:
Ah, the original hasn’t landed yet, I see. I guess you’ll want to rebase this afterwards.